steve-chae-multiple-choice-questions.md

```diff
@@ -0,0 +1,96 @@
+<!--
+	Correct answer is 2.
+	
+	1 - NgModule is the name of the Decorator used for any Angular Module.
+	I feel one can easily mistake between NgModule and AppModule
+	
+	3 - while it's possible that the root module can be anything, 
+	convention is to name it AppModule and that's what our slides say. 
+	This could be a good chance to bring it up that it's convention.
+	
+	(https://angular.io/docs/ts/latest/guide/appmodule.html)
+-->
+The name for the root module should be _________.:
```

Comment:
I feel the knowledge required for this question would be mostly academic in the sense the student won't need to use it in real life (much) because 

1.  The name of the root module would be automatically setup by angular-cli
2.  If the student has to set it up manually, it would be an once-in-a-project affair.  

Maybe a better question could be `If I want to write a custom UI Component for my angular2 project, which field of the @NgModule decorator should I put the class in?` -> Correct answer is 'declarations'